docs/resources/built-in-decorators.md

```diff
@@ -65,3 +65,32 @@ lint:
       tagNames:
         pet: ./my-custom-description.md
 ```
+
+### hide-internals
```

Comment:
The opening issue calls for it to be `hide-x-internal` but I wonder if `remove-x-internal` would be more clear.

Reply:
I mention below I see this is more flexible by allowing another token instead of `x-internal` for evaluation of what to remove.

Reply:
@adamaltman 
`remove-x-internal` sounds good, for now using `hide-x-internal`.
@RomanHotsiy what do you think?

Reply:
`remove-x-internal` sounds better 👍

Reply:
done